src/main/riddl/Order/Order.riddl

```diff
@@ -4,16 +4,16 @@ context OrderContext is {
   type LineItem is {
     product: SKU,
     quantity: Number,
-    unitPrice: Number,
     lineTotal: Number
   }
 
   type OrderInfo is {
+    order_id: OrderId,
     lineItems: LineItem+,
-    specialInstructions: String,
-    orderTotal: Number
```

Comment:
Yup. good with this change. But we have to remember that we need to calculate the line total for each line item and then sum all the lineTotals from each lineItem in lineItems.

Reply:
@miguelTaningcoYoppworks - which comment above?

Reply:
Left a direct message to Reid asking how calculated fields must be documented on riddl or how fields only present internally must be documented.